### PR TITLE
ASMBLY-1152 Add separate configs for a d500f positioner

### PR DESCRIPTION
**Pull request by pclay614**
[ASMBLY-1152](https://pathrobotics.atlassian.net/browse/ASMBLY-1152)

# Changes
* Add D500F config files
    * Axis 1 and axis 2 and reversed from the default D500B model.

# Testing
Confirmed that in sim, when using:
- D500B (default)
    - 15 degree tilt (tilted right)
    - 30 degree turn (turned clockwise)
- D500F
    - 15 degree tilt (tilted left)
    - 30 degree turn (turned counterclockwise)

[ASMBLY-1152]: https://pathrobotics.atlassian.net/browse/ASMBLY-1152?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: motoman_motopos_d500_support/urdf/motopos_d500.xacro
Comment (Doomerdinger) [inline motoman_motopos_d500_support/urdf/motopos_d500.xacro:1]: Please revert changes to this file, except the comment mentioning that it is the D500B.

File: motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro
Comment (Doomerdinger) [inline motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro:1]: Please revert changes to this file, except the comment mentioning that it is the D500B

File: motoman_motopos_d500_support/urdf/motopos_d500.xacro [hunk 1]
```diff
@@ -1,12 +1,13 @@
-<?xml version="1.0" ?>
+<?xml version="1.0"?>
+<!-- D500B model -->
```
Comment (Doomerdinger) [inline motoman_motopos_d500_support/urdf/motopos_d500.xacro]: Move this inside the `robot` tag.

File: motoman_motopos_d500_support/urdf/motopos_d500f_macro.xacro
Comment (Doomerdinger) [inline motoman_motopos_d500_support/urdf/motopos_d500f_macro.xacro:1]: I'd generally prefer this to be exactly the same as the other file except the relevant joint invert, but this is fine for now.

**Review by Doomerdinger — CHANGES_REQUESTED** [review]

File: motoman_motopos_d500_support/urdf/motopos_d500.xacro
Comment (pclay614) [reply motoman_motopos_d500_support/urdf/motopos_d500.xacro:1]: I would like to keep the formatting changes to conform to our code style guidelines.

File: motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro
Comment (pclay614) [reply motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro:1]: would like to use the VS XML formatter [Redhat] im using. Can discuss XML formatting style guidelines if you don't feel ours match what they should be.

File: motoman_motopos_d500_support/urdf/motopos_d500f_macro.xacro
Comment (pclay614) [reply motoman_motopos_d500_support/urdf/motopos_d500f_macro.xacro:1]: same as above.

File: motoman_motopos_d500_support/urdf/motopos_d500.xacro
Comment (Doomerdinger) [reply motoman_motopos_d500_support/urdf/motopos_d500.xacro:1]: This repo is not entirely our own.
It has been pulled into the upstream in the past, and we pull in updates as well.
I'd prefer to not to format existing files to reduce conflicts and make future pulls into the upstream easier.

File: motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro
Comment (pclay614) [reply motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro:1]: as discussed, reverted formatting changes as this is a Fork of a public repo. I put a blurb in our code style about that.

**Review by Doomerdinger — APPROVED** [review]

File: motoman_motopos_d500_support/urdf/motopos_d500.xacro
Comment (pclay614) [reply motoman_motopos_d500_support/urdf/motopos_d500.xacro:1]: as discussed, i agree.

**Pull request merged**